internal/processors/rocky/rockyprocessor_test.go

```diff
@@ -0,0 +1,35 @@
+package rocky_test
+
+import (
+	"context"
+	_ "embed"
+	"strings"
+	"testing"
+
+	"github.com/metal-toolbox/audito-maldito/internal/processors/rocky"
+	"github.com/stretchr/testify/assert"
+)
+
+//go:embed test_files/secure.log
```

Comment:
The convention for Go test data is to store it in a directory named `testdata` (see: `go help test | grep testdata`). Please move/rename the directory.